src/Utils/StrTree.dfy

```diff
@@ -11,11 +11,11 @@ module Utils.StrTree {
   {
     function method ToString() : string {
       match this
-        case Str(s) => s
```

Comment:
You don't indent `case`?

Reply:
Not when there are no braces { } around them.